course_gather/settings/base.py

```diff
@@ -92,18 +96,6 @@
 
 WSGI_APPLICATION = 'course_gather.wsgi.application'
 
-
```

Comment:
Should also add other database services in local_dev and dev

Reply:
By other database services you mean enabling Postgres for dev as well?

Reply:
Yep. Were we keeping sqlite for local_dev?

Reply:
What about leaving SQLite as the default for dev, and then you can override that if you want in local_dev?

Otherwise, if we make Postgres the default for dev and someone is trying to quickly set this up for the first time they either have to create a local_dev or set up a local Postgres server.

Reply:
Just now noticed you deleted local_dev, was there any reason for that in particular? 

The point of it was just for this. To have a settings that a developer can select when they don't want to setup whatever DB engine we throw in and just keep using sqlite

I'm not totally sure what you meant by default in dev and then override in local dev seeing as those two have no relation other than that they both inherit base settings.

Reply:
Whoops, thanks for catching that. I need to get out of the habit of just doing whatever I want to files that are grayed-out (editor + `.gitignore`).

Added SQLite to local dev, and made Postgres the default in dev (with env vars): 9248389.

Do you think local_dev should inherit from dev instead of base by default (so debug toolbar and CORS are enabled)?

Reply:
I think just leaving it as inheriting from base and copying over changes in dev to `local_dev` will be just fine.

Reply:
Sounds good, should be ready to merge now.